utils/networks.ts

```diff
@@ -135,3 +138,15 @@ export const getEtherscanUrl = (txHashOrAddress: string, chainId: number) => {
     return `${networkPrefix}/${group}/${txHashOrAddress}`;
   }
 };
+
+export const buildBaseToken = (baseToken: BaseToken, chainId: number) => {
+  return {
+    chainId,
+    symbol: baseToken.symbol,
+    address: ethAddress,
+    decimals: 18,
+    name: baseToken.name,
+    logoURI: baseToken.logoURI,
+    balance: BigNumber.from(0), // TODO: Add balance fetch
```

Comment:
It is okay if we default this to 0 right? It seems to be overridden in the hook, but want to call it out here.

Reply:
here's the part hook where balance is set:

https://github.com/ScopeLift/railgun-shielder/blob/main/hooks/useTokenBalances.tsx#L42-L46

Yeah, I think 0 makes sense as a default. Note that currently, i'm not sure if our useTokenBalances "native" balance calculation is appropriate for all networks. I'll make an issue to track.